com.beatwaves.responsible/Runtime/Bdd/BddTest.Unity.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using Responsible.Unity;
+using static Responsible.Bdd.Keywords;
+
+namespace Responsible.Bdd
+{
+	/// <summary>
+	/// Base class for BDD-style tests using attributes.
+	/// </summary>
+	/// <remarks>
+	/// This is Unity-only, because we depend on NUnit in the attributes,
+	/// and don't depend on NUnit for the vanilla C# version of Responsible.
+	/// A non-Unity version might be implemented later.
+	/// </remarks>
+	/// <seealso cref="FeatureAttribute"/>
+	/// <seealso cref="ScenarioAttribute"/>
+	public abstract class BddTest
```

Comment:
This class is required because AFAIK, NUnit doesn't really support set ups, tear downs etc, if we make it execute a method in any other class. Some discussion [here](https://github.com/nunit/nunit/issues/616). Also, Unity makes this even more complicated with `IEnumerableTestMethodCommand` being internal.